war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -325,7 +351,11 @@ public void addUserToOrganization(final String organizationId, final String user
     *       When user already exists in project
     */
    public void addUserToOrganization(final String organizationId, final String userName) throws UserAlreadyExistsException {
-      addUserToOrganization(organizationId, userName, readDefaultRoles(organizationId));
+      List<String> roles = readDefaultRoles(organizationId);
+      if (roles == null) {
+         roles = Collections.emptyList();
```

Comment:
Hmm, we should probably put an empty list there when creating an organization. But also return an empty list in `readDefaultRoles` method instead of `null`.